docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -103,13 +104,17 @@ The AI Assistant uses {ml-docs}/ml-nlp-elser.html[ELSER], Elastic's semantic sea
 
 NOTE: Your AI provider may collect telemetry when using the AI Assistant. Contact your AI provider for information on how data is collected.
 
-You can add information to the knowledge base by asking the AI Assistant to remember something while chatting (for example, "remember this for next time"). The assistant will create a summary of the information and add it to the knowledge base.
+External data can be added to the knowledge base through the following methods:
```

Comment:
This is an example of passive voice. Here's an article explaining what that is: https://www.grammarly.com/blog/sentences/active-vs-passive-voice/. Try to use active voice instead of passive whenever possible.

I recommend updating this to something like:

```suggestion
Add external data to the knowledge base with one or more of the following methods:
```

Reply:
I missed this one. I will review it soon :)